circle.yml

```diff
@@ -15,11 +14,14 @@ dependencies:
 test:
   override:
     - bundle exec rake
-    - docker build -t=$registry_root/$image_name:b$CIRCLE_BUILD_NUM .
+    - docker build -t=$registry_root/$CIRCLE_PROJECT_REPONAME:b$CIRCLE_BUILD_NUM .
```

Comment:
Makes sense to me: I'll compare to CLI & update. Definitely agree about capitalization: I wasn't sure if there was a reason they weren't already that way.